### DISCUSSION TITLE
How to get notified, when short link is visited

### DISCUSSION THREAD

**Discussion by angel1st** [shlinkio/shlink#2088] [Help wanted] [answered]
### Shlink version

4.0.3

### PHP version

shlinkio/shlink:stable image

### How do you serve Shlink

Docker image

### Database engine

MySQL

### Database version

8.0

### Summary

I have shlink running inside Docker container. All seems good. 
I would like to get notified, ideally via web hook, each time somebody visits a short link. I can see visits are counted and web client shows them, it would be great if I can get this information in real time via web hook.
Is this even possible?
Thanks!

**Comment by acelaya** [comment, marked as answer]
Webhooks specifically are not supported, but there're a few mechanisms you can use to subscribe for real-time updates https://shlink.io/documentation/advanced/real-time-updates/
